docs/guide/siteaccess.md

```diff
@@ -323,8 +325,8 @@ ezpublish:
 
```

Comment:
` **Example.** URI: `/something/my/content``
I think we should change `something` to `ezdemo_site` (otherwise this example does not make sense).